dynamic-plugins/wrappers/backstage-community-plugin-redhat-argocd/package.json

```diff
@@ -39,7 +39,7 @@
     "export-dynamic:clean": "run export-dynamic --clean"
   },
   "dependencies": {
-    "@backstage-community/plugin-redhat-argocd": "1.10.4",
+    "@backstage-community/plugin-redhat-argocd": "1.11.0",
```

Comment:
could you please update the `version` field in the top of this file too? That is the one which will be displayed in the UI, so lets keep that in-sync with dependancy version.

Reply:
Good catch! changed.